nbm-maven-plugin/src/test/java/org/apache/netbeans/nbm/CreateWebstartAppMojoTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.netbeans.nbm;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;
+import org.apache.maven.artifact.repository.ArtifactRepository;
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.testing.AbstractMojoTestCase;
+import static org.codehaus.plexus.PlexusTestCase.getBasedir;
+import org.junit.Assert;
+import static org.junit.Assert.assertThrows;
+import org.mockito.Mockito;
+
+/**
+ *
+ */
+public class CreateWebstartAppMojoTest extends AbstractMojoTestCase {
```

Comment:
webstart is wasted CI time IMO since it doesn't really exist anymore.

Reply:
yep maybe we should depreciate later. 
Installer kind of too if nbi removal it will be better not to generate vintage. need nbpackage in form of maven plugin 😄

Reply:
I would be tempted to deprecate that, the installers and the zip builder for a start.  Better to concentrate on the template POM and using standard Maven plugins where possible.